src/FuncHW/Program.cs

```diff
@@ -27,11 +27,18 @@ static void Main()
 
             //Func*** funcPersonIsActive = ***;
             //Func*** funcPersonPhoneNumberStartsWith7 = ***;
+            Func<Person, bool> funcPersonPhoneNumberStartsWith7 = (Person person) => 
+                person.PhoneNumber / 10000 == 7;
```

Comment:
No, that's bad to use `/ 10000`. What if appear new phone number with six symbols? e.x. 123456. 
Try to come up with something else.

Reply:
Remake check for any length of number.